src/shared/lib/resources/proposal/code-with-us.ts

```diff
@@ -2,9 +2,10 @@ import { CWUOpportunitySlim } from 'shared/lib/resources/code-with-us';
 import { FileRecord } from 'shared/lib/resources/file';
 import { Organization } from 'shared/lib/resources/organization';
 import { User } from 'shared/lib/resources/user';
-import { ADT, Id } from 'shared/lib/types';
+import { ADT, BodyWithErrors, Id } from 'shared/lib/types';
+import { ErrorTypeFrom } from 'shared/lib/validation';
 
-export enum ProposalStatus {
+export enum CWUProposalStatus {
   Draft = 'DRAFT',
   Submitted = 'SUBMITTED',
   Review = 'REVIEW',
```

Comment:
Is it possible to rename this to `UnderReview`?

Reply:
Status changes to `UnderReview` will happen automatically when a proposal deadline for an opportunity lapses. We should talk about how to handle this. Either via a daemon, or on-demand.